### DISCUSSION TITLE
tournament of the day feature

### DISCUSSION THREAD

**Discussion by flyerbram002** [ppy/osu#20464] [Ideas]
I think it'd be fun to implement something like this. It is a thing that exists in trackmania, another game which has nothing to do with osu. But i think it'd work out well. Basically you'd have a lobby each day with one map and anyone can join. Then everyone gets say ten minutes to practice set map and get the best possible accuracy or score. Then the size of the lobby would get decreased overtime after each round (one chance to play the song) there would be a few people eliminated. This could be added for all gamemodes and add another competitive element to the game for top players. 

It could already be done manually but as a game feature i think it'd be alot more fun.

**Comment by peppy** [comment]
We do plan on having event playlists running, similar to the spotlights ones we're already running. Make sure to check out spotlights if you haven't already (it's not daily but still a similar idea).
